tests/Cargo.toml

```diff
@@ -43,7 +43,7 @@ serde                    = { version = "1", features = ["derive"] }
 serde_json               = "1.0"
 serde_yaml               = "0.9"
 subnetworks-assignations = { workspace = true }
-tempfile                 = "3.6"
+tempfile                 = "3"
```

Comment:
Why this change?

Reply:
> Why this change?

Hi, keep() function cannot be called on tempfile versions earlier than 3.20 - latest.

Reply:
It would have been resolved locally with `cargo update`, but I think it's ok to have it bit more relaxed in tests.